widgetsnbextension/package.json

```diff
@@ -7,8 +7,9 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack",
-    "update:widgets": "rimraf node_modules/jupyter-js-widgets; npm install file:../jupyter-js-widgets; npm run build",
-    "prepublish": "npm run build"
+    "update": "rimraf node_modules/jupyter-js-widgets; npm install file:../jupyter-js-widgets; npm run build",
+    "prepublish": "npm run build",
+    "nbextension": "pip install -v -e . && jupyter nbextension install --py --symlink --user widgetsnbextension && jupyter nbextension enable --py --user widgetsnbextension"
```

Comment:
Since we probably want to separate this for good sometimes, we cannot assume that widgetsnbextension is present...